### PR TITLE
Remove FMEA

### PR DESCRIPTION
**Pull request by ammar92**
### Changes
This PR removes the FMEA app in Rocky as it's no longer used. Migrations file have been created and should be removed once we're confident everyone has migrated.

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by underdarknl** [comment]
rocky/assets/css/components/fmea.scss can also be removed, I think.
This would also mean the import for that file in rocky/assets/css/main.scss can be removed:

`@import "components/fmea";`

and these two lines in the rocky manifest:
```
recursive-include fmea/templates *.html
recursive-include fmea/templates *.txt
```

**Comment by underdarknl** [comment]
[[rocky/pyproject.toml](https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/rocky/pyproject.toml#L97)](https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/rocky/pyproject.toml#L97)

https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/.pre-commit-config.yaml#L88

**Comment by ammar92** [comment]
> rocky/assets/css/components/fmea.scss can also be removed, I think. This would also mean the import for that file in rocky/assets/css/main.scss can be removed:
> 
> `@import "components/fmea";`
> 
> and these two lines in the rocky manifest:
> 
> ```
> recursive-include fmea/templates *.html
> recursive-include fmea/templates *.txt
> ```

Good one. Forgot this one because I only grep'd on .py and .html files. Will fix

File: octopoes/tests/robot/05_bits.robot [hunk 1]
```diff
@@ -15,7 +15,15 @@ Bit With Scan Level 1
...
+    ...    Object List Should Contain
+    ...    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
     Declare Scan Profile    Hostname|internet|example.com    0
     Await Sync
-    Wait Until Keyword Succeeds    60s    1s    Object List Should Not Contain    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
+    Wait Until Keyword Succeeds
+    ...    60s
+    ...    1s
+    ...    Object List Should Not Contain
+    ...    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
```
[the remark this answers is not in the document]
Comment (ammar92) [reply octopoes/tests/robot/05_bits.robot:29]: For some reason it was changed by a pre-commit hook after I merged main. Ultimately I would still have to include this change if I want to make the CI happy

**Comment by ammar92** [comment]
> [[rocky/pyproject.toml](https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/rocky/pyproject.toml#L97)](
> 
> https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/rocky/pyproject.toml#L97
> 
> )
> https://github.com/minvws/nl-kat-coordination/blob/1dc808938493ade73623067b048ea6ef61abbb32/.pre-commit-config.yaml#L88

This was left on purpose since there are still migration files available that should be excluded by the hook.

File: rocky/rocky/settings.py [hunk 2]
```diff
@@ -142,7 +142,6 @@
     "two_factor",
     "account",
     "tools",
-    "fmea",
```
[the remark this answers is not in the document]
Comment (ammar92) [reply rocky/rocky/settings.py:145]: You're right, just confirmed this with `django-admin showmigrations`

**Review by dekkers — APPROVED** [review]

**Pull request merged**